Subject: Disable tests which require data not present in the crate download

The avif and mp4a files are in the Github repo but not in the crate
---

--- a/tests/public.rs
+++ b/tests/public.rs
@@ -228,6 +228,7 @@ static AUDIO_AMRWB_3GP: &str = "tests/amr_wb_1f.3gp";
 // "ffmpeg -i [input file] -f mp4 -c:v mpeg4 -vf scale=176x144 -frames:v 1 -an output.mp4"
 static VIDEO_MP4V_MP4: &str = "tests/bbb_sunflower_QCIF_30fps_mp4v_noaudio_1f.mp4";
 
+/*
 // Adapted from https://github.com/GuillaumeGomez/audio-video-metadata/blob/9dff40f565af71d5502e03a2e78ae63df95cfd40/src/metadata.rs#L53
 #[test]
 fn public_api() {
@@ -1511,3 +1512,4 @@ fn public_video_mp4v() {
         };
     }
 }
+*/
